types/node/v8/index.d.ts

```diff
@@ -5575,6 +5576,48 @@ declare module "util" {
     export namespace promisify {
         const custom: symbol;
     }
+
+    export interface TextDecodeOptions {
+        fatal?: boolean;
+        ignoreBOM?: boolean;
+    }
+    export interface TextDecoderOptions {
+        fatal?: boolean;
+        ignoreBOM?: boolean;
+    }
+    interface TextDecoder {
+        readonly encoding: string;
```

Comment:
In case a `class` can be used instead an `interface` the read only properties should be replaced by getters.